assembler/manual.html

```diff
@@ -382,13 +382,15 @@ <h3>3.1 SPAdes input</h3>
 
 <p> Notes:
     <ul>
-        <li>It is strongly suggested to provide multiple paired-end and mate-pair libraries according to their insert size (from smallest to longest).
+        <li>It is strongly suggested to provide multiple paired-end and mate-pair libraries <b>according to their insert size (from smallest to longest)</b>.
         <li>It is not recommended to run SPAdes on PacBio reads with low coverage (less than 5).</li>
         <li>We suggest not to run SPAdes on PacBio reads for large genomes.</li>
         <li>SPAdes accepts gzip-compressed files.</li>
     </ul>
 
 <h4>Read-pair libraries</h4>
+<p>
+    If you already have forward and reverse reads available in two separate files, use these as input. Otherwise spades will split the interleaved FASTQ file using its pythonic code into two files anyway, and complain if there are any singletons included.
```

Comment:
Why one would be interest in emphasize of python code?

Reply:
Because it is inefficient and there are other tools doing a faster job. It is good it is available for general use but if one cares about speed&IO for a large project than certainly this is a valid comment. Would I have known that I would serve spades with the two separate files immediately. I was lazy to do that because it needed more typing on the commandline.